settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
We didn't include the actual plugin project here. It's a separate Gradle project, linked into buildSrc of the root project via some magic in the buildSrc's `build.gradle`.